src/assets/config/appconfig.json

```
@@ -50,6 +50,7 @@
     "MaxEncLimit": "250",
     "MaxEncSelectionLimit":"100",
     "apiUrl": "",
-    "apiScope": ""
+    "apiScope": "",
+    "avgSizeofENC":"0.3"
```

Please specify units in the name of the app setting `avgSizeofENC`. Please change to `avgSizeofENCinMB`
```suggestion
    "avgSizeofENCinMB":"0.3"
```

---

resolved by renaming